es6.js

```diff
@@ -1,19 +1,49 @@
 module.exports = {
-  'parser': 'babel-eslint',
-  'env': {
-    'es6': true
+  "plugins": ["babel"],
+  "parser": "babel-eslint",
+  "env": {
+    "es6": true
   },
-  'ecmaFeatures': {
-    'modules': true,
-    'blockBindings': true,
-    'objectLiteralShorthandMethods': true,
-    'generators': true,
-    'destructuring': true,
-    'arrowFunctions': true,
-    'restParams': true,
-    'spread': true,
-    'templateStrings': true,
-    'objectLiteralComputedProperties': true
+  "ecmaFeatures": {
+    "modules": true,
+    "blockBindings": true,
+    "objectLiteralShorthandMethods": true,
+    "generators": true,
+    "destructuring": true,
+    "arrowFunctions": true,
+    "restParams": true,
+    "spread": true,
+    "templateStrings": true,
+    "objectLiteralComputedProperties": true
   },
```

Comment:
Yup, these can be removed except `modules` because you can enable es6 features by:

```js
"env": {
  "es6": true
}
```